builtins/amp-img.js

```diff
@@ -299,7 +299,12 @@ export class AmpImg extends BaseElement {
       return;
     }
 
-    // A V1 amp-img loads and reloads automatically.
+    // A V1 amp-img loads and reloads automatically in `mountCallback`.
+    this.setReadyState(ReadyState.LOADING);
```

Comment:
This is a really subtle bug if it's missed. If you don't call this in `buildCallback`, then the element won't be treated as a loadable element and be immediately marked as `complete`

We should probably add this as another static method, and then throw if you try to call `setReadyState` and you didn't mark yourself as loadable.

Reply:
Yeah. I've been thinking about this as well. I think a static prop is also in order.

Reply:
Similar to bento components? A `loadable` prop?

Reply:
Yeah.  Prob. I'll see what I can work out here.

Reply:
Done